docs/reference/slate-react/plugins.md

```diff
@@ -109,7 +109,15 @@ This handler is called whenever the native DOM selection changes.
 
 `Function renderEditor(props: Object, next: Function) => ReactNode|Void`
 
-The `renderEditor` property allows you to define higher-order-component-like behavior. It is passed all of the properties of the editor, including `props.children`. You can then choose to wrap the existing `children` in any custom elements or proxy the properties however you choose. This can be useful for rendering toolbars, styling the editor, rendering validation, etc. Remember that the `renderEditor` function has to render `props.children` for editor's content to render.
+The `renderEditor` property allows you to define higher-order-component-like behavior. It is passed all of the properties of the editor, including `children`, which you can access as `next()`. You can then choose to wrap the existing `children` in any custom elements or proxy the properties however you choose. This can be useful for rendering toolbars, styling the editor, rendering validation, etc. Remember that the `renderEditor` function has to render `children` for editor's content to render. For example:
+```js
+renderEditor: ({ editor }, next) => (
```

Comment:
Can you change this to be `(props, next)` without the destructuring to not confuse people?

Reply:
As far as I see in https://github.com/ianstormtaylor/slate/blob/master/examples/plugins/word-count.js#L14 the second parameter is editor and the thrid is next
```suggestion
renderEditor: (props, editor, next) => (
```

Reply:
@valorize probably it has changed recently, during writing this PR next was passed as 2nd argument

Reply:
@valoricDe @ianstormtaylor I updated PR to match to (props, editor, next) function signature

Reply:
@klis87 👍 But sadly the build step is failing. Could you check what the output of `prettier  '**/*.{md,json,css}'` for the file `docs/reference/slate-react/plugins.md` is?

Reply:
@valoricDe I wasn't aware prettier is used for md files, I pushed the prettified version and now build passes